opendj-server-example-plugin/pom.xml

```diff
@@ -38,7 +38,7 @@
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
-      <version>3.3.2</version>
+      <version>3.17.0</version>
```

Comment:
This line could be removed. Dependency version is already defined in OpenIdentityPlatform/commons project